script.js

```diff
@@ -0,0 +1,183 @@
+let allTasks = JSON.parse(localStorage.getItem('task')) || [];
+let activeEditTask = null;
+let currentText = "";
+const url = 'http://localhost:8000';
+const fetchHeaders = {
+  'Content-Type': 'application/json;charset=utf-8',
+  'Access-Control-Allow-Origin': '*'
+}
+
+window.onload = async  () => { // проверяем изменения 
+  const input = document.getElementById("add-task");
+  input.addEventListener('change', updateValue);
+  const resp =  await fetch(`${url}/allTask`, {
```

Comment:
/allTask

you have other routes on back
How does the program work with different routes?

Reply:
add a try/catch error handler